tests/Unit/inc/common/rocketCleanCachePostOnStatusChange.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\common;
+
+use Brain\Monkey\Functions;
+
+
+/**
+ * Test class covering ::rocket_clean_post_cache_on_status_change
+ * @uses   ::rocket_clean_files
+ *
+ * @group  Common
+ * @group  Purge
+ */
+class Test_RocketCleanCachePostOnStatusChange extends TestCase {
```

Comment:
```suggestion
class TestRocketCleanPostCacheOnStatusChange extends TestCase {
```

Reply:
This throw an error, **Class rocketCleanCachePostOnStatusChange could not be found in /var/www/html/wp-content/plugins/wp-rocket/tests/Unit/inc/common/rocketCleanCachePostOnStatusChange.php**

Reply:
You need to update the class name and the filename too. Notice that rocketClean**CachePost**OnStatusChange doesn't match rocket_clean_**post_cache**_on_status_change.

You'll also need to update the fixtures filename